Dockerfile-ubuntu

```diff
@@ -2,8 +2,8 @@ FROM --platform=linux ubuntu:22.04 as base
 ARG BUILDARCH
 
 # Change your versions here
-ENV GO_VERSION=1.18.3
-ENV IGNITE_VERSION=0.22.1
+ENV GO_VERSION=1.21.3
+ENV IGNITE_VERSION=28.4.0
```

Comment:
This is a much bigger change than just this as it has downstream effects on the Cosmos SDK version and the files produced.
The whole exercise needs updating, not just his file.